apps/client/src/features/app-settings/panel/client-control-panel/ClientList.tsx

```diff
@@ -0,0 +1,205 @@
+                  <Badge variant='outline' size='sx'>
+                    {id}
+                  </Badge>
+                  <Badge hidden={!isCurrent} variant='outline' colorScheme='yellow' size='sx'>
+                    self
+                  </Badge>
+                  {name}
+                </td>
+                <td className={style.actionButtons}>
+                  <Button
+                    size='xs'
+                    className={`${identify ? style.blink : ''}`}
+                    variant={identify ? 'ontime-filled' : 'ontime-subtle'}
+                    onClick={() => {
+                      setIdentify({ target: id, state: !identify });
+                    }}
+                    isActive
+                  >
+                    Identify
+                  </Button>
+                  <Button size='xs' variant='ontime-subtle' onClick={() => openRename(id)}>
+                    Rename
+                  </Button>
+
+                  <ButtonGroup size='xs' isAttached variant='ontime-subtle'>
+                    <Button isLoading={isRedirecting} onClick={() => openRedirect(id)}>
+                      Redirect
+                    </Button>
+                    {isRedirecting && (
+                      <IconButton
+                        aria-label='Cancel the redirect'
+                        icon={<IoClose />}
+                        onClick={() => setRedirect({ target: id, path: '' })}
+                      />
+                    )}
+                  </ButtonGroup>
+                </td>
+              </tr>
+            );
+          })}
+        </tbody>
+      </Panel.Table>
+    </>
+  );
+}
+
+function RedirectModal(props: {
+  onClose: () => void;
+  isOpen: boolean;
+  id: string;
+  clients: Clients;
+  onSubmit: (path: string) => void;
+}) {
+  const { onClose, isOpen, id, clients, onSubmit } = props;
+  const [path, setPath] = useState('');
+
+  return (
+    <Modal isOpen={isOpen} onClose={onClose} variant='ontime'>
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader>Redirect {clients[id]?.name}</ModalHeader>
+        <ModalCloseButton />
+        <ModalBody>
+          <div>Redirect to a new URL</div>
+          <InputGroup variant='ontime-filled' size='md'>
+            {/* TODO: better description */}
+            <InputLeftAddon>ontime:port/</InputLeftAddon>
+            <Input placeholder='newpath?and=params' value={path} onChange={(event) => setPath(event.target.value)} />
+          </InputGroup>
+        </ModalBody>
+        <ModalFooter>
+          <div className={style.buttonSection}>
+            <Button size='md' variant='ontime-subtle' onClick={onClose}>
+              Cancel
+            </Button>
+            <Button size='md' variant='ontime-filled' onClick={() => onSubmit(path)}>
+              Submit
+            </Button>
+          </div>
+        </ModalFooter>
+      </ModalContent>
+    </Modal>
+  );
+}
+
+function RenameModal(props: {
```

Comment:
question: why dont we use the existing `RenameClientModal`?

Reply:
the existig modal sets the local cliet name
should I rewirte that?